test/Swashbuckle.AspNetCore.SwaggerGen.Test/SchemaGenerator/JsonSerializerSchemaGeneratorTests.cs

```diff
@@ -770,6 +770,17 @@ public void GenerateSchema_HonorsSerializerOption_PropertyNamingPolicy()
             Assert.Equal(expectedDefaultAsJson, propertySchema.Default.ToJson());
         }
 
+        [Fact]
+        public void GenerateSchema_HonorsEnumDictionaryKeys_StringEnumConverter()
+        {
+            var subject = Subject();
+            var schemaRepository = new SchemaRepository();
+
+            var referenceSchema = subject.GenerateSchema(typeof(Dictionary<IntEnum, string>), schemaRepository);
+
+            Assert.Equal(typeof(IntEnum).GetEnumValues().Length, referenceSchema.Properties.Count);
+        }
+
```

Comment:
I just took a look, and we don't have an explicit test in this file for a string enum dictionary. Could you add one that's effectively the same as this test for for a dictionary wanting a string enum as the key?